tests/workflows/basic_conditional_branch/tests/test_workflow.py

```diff
@@ -54,12 +54,14 @@ def test_stream_workflow__verify_invoked_ports():
     # WHEN we stream the Workflow
     stream = workflow.stream(
         inputs=Inputs(value=True),
-        event_filter=root_workflow_event_filter,
+        event_filter=all_workflow_event_filter,
```

Comment:
[blocking] why did we change these tests? would like to keep all of these to the most restrictive filter possible to still communicate the point, which in this case is still the root filter

Reply:
these tests were breaking with the recent filter changes 🤔  change it back and update the test though.

Reply:
well, basically the test is testing the node events - but those don't get returned with the root event filter

Reply:
> well, basically the test is testing the node events - but those don't get returned with the root event filter

I would expect the node events _to_ come back with the root event filter. It's likely not bc of this equality here needs to be ubdated now that `parent.workflow_definition` is not a class: https://github.com/vellum-ai/vellum-python-sdks/blob/main/src/vellum/workflows/workflows/event_filters.py#L47

Reply:
oooooo, I'll add it in so I can get these tests passing with the root_event

Reply:
thanks for catching that! 🔥